src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -472,18 +475,6 @@ export default {
 			return 'span'
 		},
 
-		inputProps() {
-			if (this.isButtonType) {
-				return null
-			}
-			return {
-				checked: this.isChecked,
-				indeterminate: this.indeterminate,
-				required: this.required,
-				name: this.name,
```

Comment:
> 'indeterminate.prop': this.indeterminate,

This key in a bind object doesn't work as `.prop` modifier and sets literally `indeterminate.prop="true"` attribute.

Reply:
I'm not sure it is possible to specify that you need `prop` modifier in `v-bind` at all... Only in render function via `domProps`

Reply:
> Why not this?
> We still need to ot use thoe properties on an input[type=button], no?

We don't have `input[tybe=button]` here, only `button[type=button]`. But anyway, all these attributes are not valid for a button (except `name`, but it is not needed).

Reply:
> > 'indeterminate.prop': this.indeterminate,
> 
> This key in a bind object doesn't work as `.prop` modifier and sets literally `indeterminate.prop="true"` attribute.

Hmmm, I've seen snipets in the past where people used sync modifiers or events key modifiers as events.
Are you sure this can't be done? :thinking:  :see_no_evil:

Reply:
> Hmmm, I've seen snipets in the past where people used sync modifiers or events key modifiers as events. Are you sure this can't be done? 🤔 🙈

Yes, did it this way as the v-bind="obj" didn't

Reply:
But nothing else as far as I know even in Vue 3...

Reply:
Alright, thanks for double checking, I was really not sure :)